src/main/javascript/view/partials/signature.js

```diff
@@ -573,7 +573,13 @@ SwaggerUi.partials.signature = (function () {
       value = _.cloneDeep(value);
       var example = value.examples;
       if (_.isString(example)) {
-        example = jsyaml.safeLoad(example);
+        try {
+          example = jsyaml.safeLoad(example);
```

Comment:
@gilbode I think we need the second `if` statement in that createJSONSample function to be an `else if` - otherwise both fire when value.examples is non-null and it's a race.